src/pudl/scripts/dbt_helper.py

```diff
@@ -193,13 +179,13 @@ def add_column_tests(
         return schema
 
     @classmethod
-    def from_table_name(cls, table_name: str, partition_column: str) -> "DbtSchema":
+    def from_table_name(cls, table_name: str) -> "DbtSchema":
         """Construct configuration defining table from PUDL metadata."""
         return cls(
             sources=[
                 DbtSource(
                     version=2,
```

Comment:
It doesn't look like `DbtSource` has a `version` attribute, but `DbtSchema` does.

Reply:
I want to add that to a list of class cleanup changes in a separate PR if that makes sense to you